src/discussions/post-comments/PostCommentsView.test.jsx

```diff
@@ -505,6 +566,66 @@ describe('ThreadView', () => {
         .not
         .toBeInTheDocument();
     });
+
+    it('pressing load more button will load next page of comments', async () => {
+      await waitFor(() => renderComponent(discussionPostId));
```

Comment:
I think we don't need waitFor here. recheck it please
await renderComponent(discussionPostId);

Reply:
without waitFor test case fails in PR